src/service/adminService.js

```
@@ -2322,34 +2322,132 @@ const adminService = {
+              totalResults,
+              totalPages,
+              currentPage: page,
+              limit,
+              hasNextPage: page < totalPages,
+              hasPreviousPage: page > 1,
+            },
+          };
         } catch (error) {
-            throw error;
+          throw error;
         }
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)